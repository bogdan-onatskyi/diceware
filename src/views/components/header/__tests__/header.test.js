import React from 'react';

import Header from '../header';

describe("Header component", () => {
    it("should match saved snapshot", () => {
        const wrapper = shallow(
            <Header text="Заголовок"/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    it("should have 2 props", () => {
        const propsStr = 'className text';
        const props = [];
        for (let prop in Header.propTypes) props.push(prop);
        expect(props.join(' ') === propsStr).toBe(true);
    });
});