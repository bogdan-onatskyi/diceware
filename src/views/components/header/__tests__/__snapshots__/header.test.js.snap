// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match saved snapshot 1`] = `
<header
  className="header"
>
  <h1>
    Заголовок
  </h1>
</header>
`;
